apps/server/src/shared/controller/transformer/sanitize-html.transformer.ts

```diff
@@ -153,16 +115,12 @@ const inputFormatsSanitizeConfig: Record<string, IInputFormatsConfig> = {
 
 export const getSanitizeHtmlOptions = (inputFormat?: InputFormat): IInputFormatsConfig => {
 	switch (inputFormat) {
-		case InputFormat.RICH_TEXT_SIMPLE:
-			return inputFormatsSanitizeConfig.RichTextSimple;
-		case InputFormat.RICH_TEXT:
-			return inputFormatsSanitizeConfig.RichText;
+		case InputFormat.RICH_TEXT_NEWS:
+			return inputFormatsSanitizeConfig.RichTextNews;
 		case InputFormat.RICH_TEXT_CK4:
 			return inputFormatsSanitizeConfig.RichTextCk4;
 		case InputFormat.RICH_TEXT_CK5:
 			return inputFormatsSanitizeConfig.RichTextCk5;
-		case InputFormat.RICH_TEXT_CK5_SIMPLE:
-			return inputFormatsSanitizeConfig.RichTextCk5Simple;
```

Comment:
RichTextCk5Simple was used as far as I know... but if this is not the case (no small balloon toolbar version of the editor), then it should be removed also in the InputFormat above